src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -287,7 +287,7 @@ public IEnumerable<ConsensusRule> GetRules()
                     // rules that require the store to be loaded (coinview)
                     new LoadCoinviewRule(),
                     new TransactionDuplicationActivationRule(), // implements BIP30
-                    new PowCoinViewRule() // implements BIP68, MaxSigOps and BlockReward calculation
+                    new PowCoinviewRule() // implements BIP68, MaxSigOps and BlockReward calculation
```

Comment:
You might want to also rename `PosCoinViewRule`

Reply:
Done